library/std/src/keyword_docs.rs

```diff
@@ -109,6 +109,33 @@ mod as_keyword {}
 /// println!("{result}");
 /// ```
 ///
+/// It is also possible to exit from any *labelled* block returning the value early.
+/// If no value specified `break;` returns `()`.
```

Comment:
Odd stutter here. I think it should be "If no value *is specified for*" or "*is given to*", then "it returns `()`".

Reply:
This is a verbatim restatement of a sentence from the previous paragraph:
> This is only valid with `loop` and not with any other type of loop.
> If no value is specified, `break;` returns `()`. 

> It is also possible to exit from any *labelled* block returning the value early.
>  If no value specified `break;` returns `()`.

Does a comma suffice or should the text above be rephrased too?

Reply:
The comma is the minimum for it to be valid, as far as I can tell. I do not think it is sufficient for it to be very comprehensible. So if you wish to revise both cases to be consistent, that is fine with me.